necessist/tests/necessist_db_absent/cli_input-mixed.toml

```
@@ -0,0 +1,11 @@
+args = ["--no-sqlite", "--dump-candidates", "--root=fixtures/cli_input", "fixtures/cli_input/src/some_dir", "fixtures/cli_input/src/other_dir/other_file.rs"]
```

```suggestion
args = ["--no-sqlite", "--dump-candidates", "--root=fixtures/cli_input", "fixtures/cli_input/src/some_dir", "fixtures/cli_input/src/other_dir/some_file.rs"]
```
Nit (so that the selected dir and file are the "some" ones).